web/src/app/shared/components/master-and-detail/master-and-detail.component.scss

```diff
@@ -5,10 +5,14 @@
 
   &__task-list {
     width: 500px;
+    min-width: 500px;
+    max-width: 500px;
   }
 
   &__task-details {
     flex-grow: 1;
+    min-width: calc(920px - 500px);
+    height: calc(100vh - 56px);
```

Comment:
keep in mind that defining hard coded width height may cause error on untested monitor size

Reply:
do you mean width and height?